dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/utils/WorkManagerUtils.kt

```diff
@@ -44,7 +45,7 @@ internal fun triggerUploadWorker(context: Context, internalLogger: InternalLogge
     try {
         val workManager = WorkManager.getInstance(context)
         val constraints = Constraints.Builder()
-            .setRequiredNetworkType(NetworkType.CONNECTED)
+            .setRequiredNetworkType(NetworkType.UNMETERED)
```

Comment:
should we let it go no matter what the network type is ? I am thinking in situations where the client is on ROAMING and we are sending data under the hood which might not be wise.

Reply:
Looks like `WorkManager` has a constraint of [NOT_ROAMING](https://developer.android.com/reference/androidx/work/NetworkType#NOT_ROAMING) - could we use this?

Reply:
oh good catch...we could indeed use that which is better, we care about ROAMING specifically here